lib/evss/pciu_address/response_strategy.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 require 'common/models/concerns/cache_aside'
+require 'evss/pciu_address/countries_response'
+require 'evss/pciu_address/states_response'
```

Comment:
Is there a way to dummy-proof this for someone who decides to use this class but isn't aware that they need to `require` their object type?

Reply:
@omgitsbillryan It's more of a cache aside issue (actually redis store) that the object being deserialized's class wasn't loaded, I added a spec but rspec autoloads things so it's not really covering the bug but it does cover caching.

Reply:
Gotcha, you're right about rspec autoloading.  I suppose it's the developers' responsibility to realize they need to `require` relevant classes, even though we've seen it's an easy mistake to make :wink: